app/mailers/weekly_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class WeeklyMailer < ApplicationMailer
+  default from: "precichysim@gmail.com"
+
+  def weekly
```

Comment:
Not sure if you need to name the method `weekly` if you're naming the class `WeeklyMailer`

Reply:
:eyes: solved :D